test/ir/pir/translator/CMakeLists.txt

```
@@ -4,6 +4,7 @@ file(
   "test_*.py")
 string(REPLACE ".py" "" TEST_INTERP_CASES "${TEST_INTERP_CASES}")
 
+set(DISTRIBUTED_OP_TRANSLATOR_TEST test_c_allreduce_min_translator)
 set(DISTRIBUTED_OP_TRANSLATOR_TEST test_c_reduce_min_translator)
```

```suggestion
set(DISTRIBUTED_OP_TRANSLATOR_TEST 
      test_c_allreduce_min_translator
      test_c_reduce_min_translator)
```
或者
set(DISTRIBUTED_OP_TRANSLATOR_TEST test_c_reduce_min_translator)
list(APPEND DISTRIBUTED_OP_TRANSLATION_TEST test_c_reduce_min_translator)

---

收到